NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -438,6 +449,26 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
     };
     toneStackSlider->SetActionFunction(toneStackAction);
 
+    // toggle IR on / off
+    IBSwitchControl* toggleIRactive = new IBSwitchControl(irBypassToggleArea, irSwitch, kIRToggle);
+    pGraphics->AttachControl(toggleIRactive, kIRToggle);
+    // dim IR dispaly
+    pGraphics
+      ->AttachControl(
+        new IVPanelControl(
+          irBypassDimPanel, "", style.WithDrawFrame(false).WithColor(kFG, COLOR_BLACK.WithOpacity(dimPanelOpacity))),
+        kNoTag, "NAM_OVERLAY")
+      ->Hide(this->GetParam(kIRToggle)->Value());
+
+    // Extend the ir toggle action function to set dim panel visible or not
```

Comment:
Nit: it seems that [`OnParamChangeUI`](https://github.com/sdatkinson/NeuralAmpModelerPlugin/blob/988697aa9ae5867089dc37f1d56e88411f12a630/NeuralAmpModeler/NeuralAmpModeler.cpp#L658) is the recommended way to handle this.

This does seem to work fine (so nit), but the other way is preferable.

Reply:
On second thought, I can merge this tonight and that'd make me happy. So let's do it!